tuned_lens/scripts/ingredients.py

```diff
@@ -301,8 +314,13 @@ def shard_dataset(self, dataset: Dataset) -> Dataset:
             dataset = dataset.shard(self.world_size, self.rank)
         return dataset
 
-    def init(self) -> None:
-        """Initialize distributed process group if started with elastic launch."""
+    def init(self):
+        """Initialize distributed process group if started with elastic launch.
+
+        Returns:
```

Comment:
This does not appear to actually return anything.

Reply:
fixed